spec/factories.rb

```diff
@@ -54,4 +54,12 @@
       data_url "I'm a number!"
     end
   end
+
```

Comment:
@jk1dd Did you get this done? If not I'm down to pair on it later if you'd like.